au4/src/projectscene/qml/Audacity/ProjectScene/clipsview/TracksClipsView.qml

```diff
@@ -112,5 +112,15 @@ Rectangle {
             anchors.bottom: parent.bottom
             x: playCursorController.positionX
         }
+
+        VerticalRulersPanel {
+            id: verticalRulers
+
+            height: parent.height - timeline.height
+            anchors.right: view.right
+            anchors.bottom: view.bottom
+
+            visible: tracksModel.isVerticalRulersEnabled
```

Comment:
I think this ruler should be part of the track delegate (TrackClipsItem) 
(i.e. one ruler for each track)